numba/cuda/tests/cudadrv/test_cuda_driver.py

```diff
@@ -113,6 +113,15 @@ def test_cuda_driver_stream(self):
         for i, v in enumerate(array):
             self.assertEqual(i, v)
 
+    def test_cuda_driver_occupancy(self):
+        module = self.context.create_module_ptx(self.ptx)
+        function = module.get_function('_Z10helloworldPi')
+
+        value = self.context.get_active_blocks_per_multiprocessor(function, 128, 128)
+        print('active blocks:', value)
+        def b2d(bs): return bs
+        grid, block = self.context.get_max_potential_block_size(function, b2d, 128, 128)
+        print('grid size:', grid, ', block size:', block)
```

Comment:
Is there anything we can assert on in this test?  I imagine that we can't assert on specific values for grid and block since those will depend on the local hardware.  Maybe we should at least verify that the return values are all greater than zero?